apps/alpha-test-app/src/components/form-demo/FormDemo.vue

```diff
@@ -1,26 +1,33 @@
 <script lang="ts" setup>
-import { OnyxButton, OnyxHeadline, TestInput } from "sit-onyx";
-import { ref, watch } from "vue";
-
-type FormData = {
+import {
+  OnyxButton,
+  OnyxCheckboxGroup,
+  OnyxHeadline,
+  OnyxInput,
+  OnyxRadioButtonGroup,
+  OnyxSwitch,
+  type OnyxCheckboxGroupProps,
+  type OnyxRadioButtonGroupProps,
+} from "sit-onyx";
+import { ref, watchEffect } from "vue";
+
+export type FormData = Partial<{
   defaultInput: string;
   requiredInput: string;
   minlengthInput: string;
-  maxInput: number;
```

Comment:
> you removed maxInput because we don't have a number input yet, right? but do we still keep the translations for now?

Good point, I removed the translations for now, is that fine for you?

> was it intentional not to move the TestInput into the alpha-test-app for the number example as discussed before easter?

Yes, I don't like having this TestInput lying around without any real purpose :) But if you insist, I can re-add it

> is there a kind of follow up ticket that points out including a number input here when we get it?

Probably the OnyxStepper component, right? Or should be support `type="number"` for the OnyxInput until we have a stepper?

Reply:
> Yes, I don't like having this TestInput lying around without any real purpose :) But if you insist, I can re-add it

would prefer that over removing the translations, because e.g. the korean translations are already there.
of course, if onyxStepper comes in the next sprint already, that would also be fine. I just don't know when we planned to implement it, if it will be some time, use the TestInput in the meantime

Reply:
> Probably the OnyxStepper component, right? Or should be support type="number" for the OnyxInput until we have a stepper?

should Jannick/Jonathan answer this maybe?

Reply:
https://github.com/orgs/SchwarzIT/projects/5/views/7?pane=issue&itemId=59042435